test/MUXControlsTestApp/Utilities/AxeScanTestPageAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+namespace MUXControlsTestApp
+{
```

Comment:
Why do we need a new attribute type?

Reply:
So we can easily create pages who are just for AXE testing.

Reply:
Right, but WHY? Why is it important to treat these axe tests as special unique snowflakes?

Reply:
My motivation was that AXEs tests are not something you touch often. Usually, you would dump a few isntances of a control, add the axe test attribute and add the appropriate interaction test (usually one line since we have a helper). After that, you won't touch that again since it servers little purpose beside hosting static controls.

Reply:
Thanks.